requirements.txt

```diff
@@ -1,5 +1,5 @@
 azure-datalake-store>=0.0.46,<0.1
 azure-storage-blob>=2.1.0
```

Comment:
Need to pin this to azure-storage-blob<3.0  Microsoft is currently doing a large rewrite, and the version on PyPi jumped from 2.1.0 --> 12.0.

Reply:
> the version on PyPi jumped from 2.1.0 --> 12.0.

! 
It must be six times better!